flink-libraries/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliTableResultView.java

```diff
@@ -159,6 +160,9 @@ protected void evaluate(ResultTableOperation operation, String binding) {
 			case PREV:
 				gotoPreviousPage();
 				break;
+			case FIRST:
```

Comment:
I think we don't need a `FIRST`. `GOTO` allows for the same functionality. `LAST` is a special feature to stay at the last page in streaming mode.

Reply:
From the function's point of view, that's true. I added it just for symmetry 😃.